src/main/java/potaton/Potaton.java

```diff
@@ -0,0 +1,299 @@
+package potaton;
+
+import potaton.task.Deadline;
+import potaton.task.Events;
+import potaton.task.Task;
+import potaton.task.ToDos;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Potaton {
+
+    public static final String TASK_LIST = "list";
+    public static final String TASK_TODO = "todo";
+    public static final String TASK_EVENT = "event";
+    public static final String TASK_DEADLINE = "deadline";
+    public static final String TASK_MARK = "mark";
+    public static final String TASK_UNMARK = "unmark";
+    public static final String TASK_DELETE = "delete";
+    public static final String EXIT = "bye";
+
+    public static final String LOADTASK_TODO = "T";
+    public static final String LOADTASK_EVENT = "E";
+    public static final String LOADTASK_DEADLINE = "D";
+
+    public static final String INIT_FILE_PATH = "potaton.txt";
+    public static final String FILE_MISSING_MESSAGE = "File does not exist. Creating file.";
+    public static final String FILE_FOUND_MESSAGE = "File found. Loading file into database.";
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Excellent use of [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.